src/ResourceManagement/CosmosDB/CosmosDBAccountImpl.cs

```diff
@@ -112,6 +114,7 @@ private async Task<Microsoft.Azure.Management.CosmosDB.Fluent.ICosmosDBAccount>
             }
 
             await this.privateEndpointConnections.CommitAndGetAllAsync(cancellationToken);
```

Comment:
Can this two be parallel? Otherwise, it will introduce additional latency.

Reply:
Done